src/otaclient/ota_core.py

```diff
@@ -1490,7 +1507,9 @@ def main(
                         session_id=request.session_id,
                     )
                 )
-                _allow_request_after = _now + HOLD_REQ_HANDLING_ON_ACK_REQUEST
+                _allow_request_after = (
+                    _now + HOLD_REQ_HANDLING_ON_ACK_CLIENT_UPDATE_REQUEST
+                )
```

Comment:
16 seconds was too long in the testing. Reduced it to 4 sec only in client update.